.pfnci/linux/tests/cuda130.multi.Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
        && \
     apt-get -qqy install ccache git curl && \
     apt-get -qqy --allow-change-held-packages \
-            --allow-downgrades install 'libnccl2=2.27.*+cuda13.0' 'libnccl-dev=2.27.*+cuda13.0' 'libcusparselt0=0.2.0.*' 'libcusparselt-dev=0.2.0.*'
```

Comment:
Were such old versions required?

Looks like it's looking for it in this repo: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/

Oldest is 0.4.0.7 (2023-04-28). Most recent is 0.7.1.0 (2025-02-15)

Did the main branch use older repositories to not have this issue?

Reply:
Informed by @emcastillo that cupy13 was just pinning that version and that latest cusparselt is pinned for cupy 14: https://github.com/cupy/cupy/blob/main/.pfnci/matrix.yaml#L513